src/main/kotlin/org/wycliffeassociates/otter/common/domain/ImportResourceContainer.kt

```diff
@@ -76,12 +78,14 @@ class ImportResourceContainer(
         val rc = ResourceContainer.load(container)
         val dc = rc.manifest.dublinCore
 
-        if (dc.type == "bundle" && dc.format == "text/usfm") {
-            expandResourceContainerBundle(rc)
-        }
-
-        val tree = constructContainerTree(rc)
-        return collectionRepository.importResourceContainer(rc, tree, dc.language.identifier)
+        return Single.fromCallable {
```

Comment:
didn't think about that, you're right. We might also want to just throw the first two lines in here too (loading the resource container)